packages/components/src/components/carousel/carousel.tsx

```diff
@@ -180,6 +180,7 @@ export class Carousel {
   private btnNext: HTMLPButtonPureElement;
   private paginationEl: HTMLElement;
   private slides: HTMLElement[] = [];
+  private splideSlides: HTMLElement[] = [];
```

Comment:
You can also create a private getter instead of storing it in a private member
```suggestion
  private get splideSlides(): HTMLElement[] {
    return this.splide.Components.Elements.slides;
  }
```